src/test/java/uk/gov/hmcts/reform/sandl/snlevents/actions/listingrequest/CreateListingRequestActionTest.java

```diff
@@ -65,8 +65,8 @@
     @Spy
     private HearingMapper hearingMapper;
 
-    @Spy
-    private FactsMapper factsMapper;
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
```

Comment:
Couldn't you just annotate tests like that     @Test(expected = SnlEventsException.class)
?

Reply:
This rule actually give you the ability to extract the message thrown so we can check to see the correct error has been triggered